apps/bufet/fiskalpro_parser.py

```
@@ -1,98 +1,121 @@
-    reader = csv.DictReader(io.StringIO(text), delimiter=';')
+    ws = wb.active
+    rows = ws.iter_rows(values_only=True)
 
-    required_columns = {'Artikl', 'Název', 'Množství', 'Cena celkem s DPH'}
-    if not required_columns.issubset(set(reader.fieldnames or [])):
-        missing = required_columns - set(reader.fieldnames or [])
-        raise ValueError(f"CSV neobsahuje očekávané sloupce: {', '.join(missing)}")
+    try:
+        header = next(rows)
+    except StopIteration:
+        raise ValueError("XLSX je prázdný.")
```

Ruff (0.15.20)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)